src/main/java/ee/buerokratt/ruuter/domain/Condition.java

```diff
@@ -0,0 +1,14 @@
+package ee.buerokratt.ruuter.domain;
+
+import com.fasterxml.jackson.annotation.JsonAlias;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+public class Condition {
```

Comment:
Move this file into the `steps` directory (preferably make a new `switch` directory inside the `steps` directory and move both this file and the switch step file into that)

Reply:
`switch` is not valid package name, so I put `conditional`